config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,6 +3,15 @@
         "5.1":{
             "status": "supported",
             "latest-revision": {
+                "revision": "4",
+                "release-date": "7 March 2024",
+                "cves": "CVE-2023-3446, CVE-2023-3817, CVE-2023-5678, CVE-2023-5981, CVE-2023-39615, CVE-2023-7104, CVE-2023-50387, CVE-2023-50868",
+                "golang": "",
```

Comment:
update golang image. You can get golang version used for building image from Travis build which built & pushed the image.